src/lib.rs

```diff
@@ -1576,6 +1587,7 @@ impl Default for RtcConfig {
             codec_config: CodecConfig::new_with_defaults(),
             stats_interval: Duration::from_secs(1),
             bwe: false,
+            bwe_initial_bitrate: Bitrate::kbps(1500),
```

Comment:
What is a good value here? 300 before, now 1500. Do we have a reason? Can we document the reason in code?